documentation/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
```

Comment:
Maybe add a note here saying that these commands need to be run in the `documentation` folder?

Reply:
yes, will do